### PR TITLE
Hyundai FPv2: reduce number of ECU queries

### PR DESCRIPTION
**Pull request by sshane**
Splitting out, will merge support first with no fingerprinting changes.

This was inferred by cross referencing which fw versions changed here when adding LONG https://github.com/commaai/openpilot/commit/b8cc31f888d4301b7db59ff9419db9215868a3f7 and which were changed when removing SHORT: https://github.com/commaai/openpilot/pull/22344

Assumptions were wrong, updated based on result from Optima here: https://github.com/commaai/openpilot/pull/24733

TODO:
- [ ] Figure out which request engine responds to for Hyundai, it also responds to Nissan and Mazda's requests
- [ ] Some cars' fwdRadar do not respond at all to any of our requests (service not supported)

### CODE REVIEW

File: selfdrive/car/fw_versions.py [hunk 1]
```diff
@@ -111,11 +112,13 @@ class Request:
     "hyundai",
     [HYUNDAI_VERSION_REQUEST_LONG],
     [HYUNDAI_VERSION_RESPONSE],
+    whitelist_ecus=[Ecu.fwdRadar, Ecu.esp, Ecu.engine, Ecu.eps],
```
Comment (sshane) [inline selfdrive/car/fw_versions.py:115]:
```suggestion
    whitelist_ecus=[Ecu.fwdCamera, Ecu.fwdRadar],
```

File: selfdrive/car/fw_versions.py [hunk 2]
```diff
@@ -111,11 +112,13 @@ class Request:
...
   ),
   Request(
     "hyundai",
     [HYUNDAI_VERSION_REQUEST_MULTI],
     [HYUNDAI_VERSION_RESPONSE],
+    whitelist_ecus=[Ecu.fwdCamera, Ecu.transmission],
```
Comment (sshane) [inline selfdrive/car/fw_versions.py:121]:
```suggestion
    whitelist_ecus=[Ecu.esp, Ecu.transmission, Ecu.fwdRadar],
```

**Comment by jyoung8607** [comment]
> * [ ]  Some cars' fwdRadar do not respond at all to any of our requests (service not supported)

If you get local or remote hands on one, try first sending a session change to extended (from default).

**Comment by sshane** [comment]
Checked the first segment of all qlog routes in the last 14 days, there were no instances where a VW ECU returned with a fw version that wasn't what were were expecting in this PR 🥳:

```python
Checked platforms:
{'VOLKSWAGEN PASSAT 8TH GEN': 720, 'VOLKSWAGEN TIGUAN 2ND GEN': 155, 'SKODA SUPERB 3RD GEN': 55, 'VOLKSWAGEN JETTA 7TH GEN': 135, 'VOLKSWAGEN ATLAS 1ST GEN': 65}
Checked dongles:
{'8dbccfa899c0d1b8': 550, 'c439baf36d9b6107': 150, '7291d51974eb2d1d': 55, '46e8fc71a2f4298d': 165, '4bd71e1011d03cb0': 135, 'bbc2fa573509b5ff': 5, '8d09dff0af485a2c': 65, 'c78c1972adfa10df': 5}
```

**Pull request closed without merging**